lib/markdown2.py

```diff
@@ -908,6 +908,10 @@ def _strict_tag_block_sub(self, text, html_tags_re, callback):
 
         return result
 
+    def _tag_is_closed(self, tag_name, text):
+        # super basic check if number of open tags == number of closing tags
+        return len(re.findall('<%s(?:.*?)>' % tag_name, text)) == len(re.findall('</%s>' % tag_name, text))
```

Comment:
Wonder if this could be even simpler on the left side, something as simple as even:

```
len(re.findall('<%s' % tag_name, text)) == len(re.findall('</%s>' % tag_name, text))
```

which just bails on matching the opening element's end > and middle content.

Reply:
I mainly put that there because I was concerned about incomplete tags potentially messing it up. Not sure as to how something like `<div> <div </div>` should be interpreted

Reply:
Yup good call